src/table/utils/handle-accessibility.ts

```diff
@@ -145,3 +145,16 @@ export const announceSelectionState = (
       : announce({ keys: ['SNTable.SelectionLabel.NotSelectedValue'] });
   }
 };
+
+export const handleKeyToCopyCell = (cell: Cell) => {
+  const textElement: HTMLTextAreaElement = document.createElement('textarea');
+  textElement.style.position = 'absolute';
+  textElement.style.left = '-9999px';
+  textElement.setAttribute('readonly', '');
+
+  textElement.innerText = cell.qText as string;
+  document.body.appendChild(textElement);
+  textElement.select();
+  !document.execCommand('copy') && alert('The content is not copied.');
+  document.body.removeChild(textElement);
+};
```

Comment:
This is the old version by using execCommand but then I replaced it with ClipboardAPI in the next commit.